### DISCUSSION TITLE
포스트 프로세싱 사용 가능한가요 ??

### DISCUSSION THREAD

**Discussion by lhj5709** [naverz/zepeto-studio-kor#364] [5. Q&A]
그냥 적용했을 때는 반영이 안되는 것 처럼 보이네요,,

사용가능한지, 어떤 방법이 있을지 궁금합니다

**Comment by x4cf** [comment]
아직은 사용 불가능하고 그래픽 랜더 함수가 업데이트되면 셰이더를 직접 제작하여 사용 가능합니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 포스트 프로세싱 기능이 월드 1.1.5에 반영되었습니다.
감사합니다.
